wemake_python_styleguide/visitors/ast/builtins.py

```diff
@@ -112,6 +112,29 @@ def _check_is_alphatbet(
                 ),
             )
 
+    def _is_exception(self, parent: Optional[ast.AST]) -> bool:
+        """
+        Check if the string with modulo patterns is in an exceptional situation.
+
+        Basically we have some function names in which we allow strings with
+        modulo patterns because they must have them for the functions to work
+        properly.
+        """
+        exceptions = (
+            'strftime',  # For date, time, and datetime.strftime()
+            'strptime',  # For date, time, and datetime.strptime()
+            'execute',  # For psycopg2's cur.execute()
+        )
+
+        if parent and isinstance(parent, ast.Call):
+
+            func_name = getattr(parent.func, 'attr', None)
```

Comment:
Oh, one more thing: what about `given_function_call` here? https://github.com/wemake-services/wemake-python-styleguide/blob/master/wemake_python_styleguide/logic/tree/functions.py#L13

Reply:
This method has a problem: if, let's say, `datetime.strptime()` is executed, it will check that the whole `datetime.strptime` is in the exceptions, but in case the user imports `datetime` as `dt`, or anything else, it will fail. With functions like `strftime` this is even more of a problem, because it's used in datetime objects stored in variables with names defined by the users. I can use `given_function_called` for this PR, but this would require to modify that function to check only the function name instead of the function and the modules it belongs to.

I propose creating an issue to modify its behavior, either by splitting the modules and function's string by dots and comparing only the function name, or by using `endswith` which would allow the developers to use function and modules names in the `to_check` parameter.

Reply:
@Jrryy yes, please!